streaming/streaming.go

```diff
@@ -635,24 +642,40 @@ func LaunchDataflowJob(ctx context.Context, targetProfile profiles.TargetProfile
 		inputFilePattern = inputFilePattern + "/"
 	}
 	fmt.Println("Reading files from datastream destination ", inputFilePattern)
-	var dataflowHostProjectId string
-	if dataflowCfg.HostProjectId == "" {
-		dataflowHostProjectId, _ = utils.GetProject()
-	} else {
-		dataflowHostProjectId = dataflowCfg.HostProjectId
-	}
-
-	dataflowSubnetwork := ""
 
-	// If custom network is not selected, use public IP. Typical for internal testing flow.
-	workerIpAddressConfig := dataflowpb.WorkerIPAddressConfiguration_WORKER_IP_PUBLIC
-
-	if dataflowCfg.Network != "" {
+	// Initiate runtime environment flags and overrides.
+	var (
+		dataflowProjectId        = project
+		dataflowVpcHostProjectId = project
+		gcsTemplatePath          = DEFAULT_TEMPLATE_PATH
+		dataflowSubnetwork       = ""
+		workerIpAddressConfig    = dataflowpb.WorkerIPAddressConfiguration_WORKER_IP_PUBLIC
+		dataflowUserLabels       = make(map[string]string)
```

Comment:
Didnt see user labels in the config defined above, where does the user enter them?

Reply:
THey are defined under AdditionalUserLabels, the variable scope is just this function hence used the name dataflowUserLabels.